src/Rootfs.jl

```diff
@@ -988,3 +989,17 @@ function installed_shards()
 
     return all_compiler_shards()[idx]
 end
+
+
+"""
+    manage_shards()
+Open a prompt allowing a user to selectively remove downloaded compiler shards
+"""
+function manage_shards()
+    manage_shards_menu = TerminalMenus.request("Which compiler shards should be removed?",
+                                       TerminalMenus.MultiSelectMenu(string.(installed_shards()) , pagesize = 10; charset=:ascii)
+                                       )
+
+    installed_shards()[Int.(manage_shards_menu)]
+    Pkg.remove_artifact.(BinaryBuilderBase.shard_source_artifact_hash.(not_installed[Int.(manage_shards_menu)]))
```

Comment:
I don't understand these two lines:
The first seems to just index into an array without doing anything.
The second refers to a `not_installed` variable that does not exist.

Is there some code missing here?

Reply:
Probably, I opened this up at first to see if there was any interest in this type of functionality, so I might have had some psuedocode floating around as I was sketching out. 

I switched back and forth a bit starting with remove or install new shards functionality, so I might have crossed the two. There is a (unexported) function somewhere the generates a full expansion of compilers, so I might have been using that and `installed_shards` to get the differences. (`not_installed`)

Feel free to do whatever on this